pipelines/_ingredients/base-v2.yaml

```diff
@@ -989,3 +1016,6 @@ contracts:
   - reprocessVisitImage.do_apply_flat_background_ratio == calibrateImage.do_illumination_correction
   - fgcmBuildFromIsolatedStar.doApplyWcsJacobian == False if calibrateImage.do_illumination_correction else True
   - fgcmBuildFromIsolatedStar.doApplyWcsJacobian == fgcmOutputProducts.doComposeWcsJacobian
+  - (detectCoaddPeaks.useCellCoadds == deblendCoaddFootprints.useCellCoadds == measureObjectForced.useCellCoadds ==
```

Comment:
Better to split this out into N-1 separate constraints, I think; that'll help narrow down what the problem is when one trips.  And it's just easier to read, and it doesn't make the user stop and think, "oh, does that actually work?"